File: Backend-API/Staffing_Recruting-API/Model/Jobs.cs
namespace Staffing_Recruting_API.Model
{
    public class Jobs
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public string Location { get; set; }

        public string Recruiter_ID { get; set; }

        public decimal From_Salary { get; set; }

        public decimal To_Salary { get; set; }

        public string SalaryType { get; set; }

        public string Status { get; set; }

        public DateTime CreatedAt { get; set; }
    }

    public class GetJobs
    {
        public string Title { get; set; }

        public string Description { get; set; }

        public string Location { get; set; }

        public string Recruiter_ID { get; set; }

        public decimal From_Salary { get; set; }

        public decimal To_Salary { get; set; }

        public string SalaryType { get; set; }

        public string Status { get; set; }

        public DateTime CreatedAt { get; set; }
    }

    public class AddJobsDTO
    {
        public string Title { get; set; }

        public string Description { get; set; }

        public string Location { get; set; }

        public decimal From_Salary { get; set; }

        public decimal To_Salary { get; set; }

        public string SalaryType { get; set; }

        public string Status { get; set; }

        public DateTime CreatedAt { get; set; }
    }

}

File: Frontend/Staffing-Recruting/src/Models/Jobs.ts
export interface IJobs {
    id: number;
    Title: string;
    Description: string;
    Location: string;
    Recruiter_ID: string;
    From_Salary: number;
    To_salary: number;
    SalaryType: number;
    Status: string;
    CreatedAt: string;
}

export interface IAddJobs {
    Title: string;
    Description: string;
    Location: string;
    From_Salary: number;
    To_salary: number;
    SalaryType: number;
    Status: string;
}

export interface IGetJobs {
    Id:number;
    Title: string;
    Description: string;
    Location: string;
    Recruiter_ID: string;
    From_Salary: number;
    To_Salary: number;
    SalaryType: number;
    Status: string;
    CreatedAt: string;
}